### ISSUE TITLE
Record formats are partially hidden when using normal browser width

### ISSUE THREAD

**Issue by brittag** [18F/atf-eregs#304]
[§ 478.125 Record of receipt and disposition](https://atf-eregs.apps.cloud.gov/478-125/2014-18392#478-125-a) contains three tables that display the formats required for disposition and acquisition records. When viewing this page with my normal browser width, parts of all three tables are hidden, as shown in this example screenshot:

![screen shot 2016-02-01 at 5 30 09 pm](https://cloud.githubusercontent.com/assets/391313/12736953/7a76c22e-c909-11e5-8897-71c738afb960.png)

**Comment by cmc333333** [comment]
@brittag you can scroll the tables horizontally -- we had an issue to make this more clear, but I think we canned it.

**Comment by brittag** [comment]
Weird: yesterday I wasn't seeing horizontal scrollbars (you can see there's no scrollbar in my screenshot), and today I'm seeing horizontal scrollbars.

<img width="963" alt="screen shot 2016-02-02 at 9 06 57 am" src="https://cloud.githubusercontent.com/assets/391313/12757307/5a5c41de-c98c-11e5-8fdf-f15d61a1f89e.png">

I don't know how to reproduce the earlier behavior - maybe it was just happening on the big display? I'm WFH today so I can't check that today; I'll watch out to see if it happens again.

I see there's an existing issue about the overall table display over at https://github.com/18F/atf-eregs/issues/125

**Comment by cmc333333** [comment]
I believe OSX hides the scroll bar until hovering, maybe even until scrolling. Shall we say this issue is specifically around indicating that horizontal scrolling is possible?

**Comment by brittag** [comment]
Yes, consistently indicating that horizontal scrolling is possible is the key part.

Today I'm seeing the horizontal scrollbar at all times (even when not hovering over the area), which is fine - and yesterday I wasn't seeing the horizontal scrollbar even after hovering. So I'm mostly just wondering why I saw that weird behavior yesterday, and whether other people might get it sometimes too.
